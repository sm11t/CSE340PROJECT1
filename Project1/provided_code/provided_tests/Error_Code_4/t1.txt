TASKS
1 2
POLY
    F1 = x + 2;
    F2(X,Y,Z) = X + Y + Z;
EXECUTE
    a = F2(F1(3),F1(4,5));
    OUTPUT a;
    b = F2(
        3,
        4,
        F1(7,F1(8,9)),
        6
    );
INPUTS
1 2 3 4 5