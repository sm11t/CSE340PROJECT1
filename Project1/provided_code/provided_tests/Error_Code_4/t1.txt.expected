Semantic Error Code 4: 7 7 9 12 12 
